### DISCUSSION TITLE
Google Indexed Wrong Meta Desciption

### DISCUSSION THREAD

**Discussion by rizkiandrianto** [gilbarbara/react-joyride#615] [Q&A] [answered]
Hey, I just want to share the problem to everybody using `react-joyride`.
We are facing an issue, which is caused Google display wrong desc about our site.

We have a website, like this:
![image](https://user-images.githubusercontent.com/2457820/101727147-826cf580-3ae6-11eb-9adf-add65d23b905.png)

But when I search using google search, it appears like this:
![image](https://user-images.githubusercontent.com/2457820/101727126-784af700-3ae6-11eb-9af9-d52ce69963f9.png)

Seems like the joyride replacing my description, but it's not.

Anybody can tell me how the google crawler works for this, and how to avoid this?

Thanks !

Edit: 
I use Next JS (Server Side Rendered Page)

**Comment by gilbarbara** [comment, marked as answer]
Hey @rizkiandrianto 

There's no secret to how the Google crawler works. It indexes whatever it is on the rendered markup.
And this library doesn't run on the server-side since it needs the DOM, so there's no way for it to be outputting anything at that stage...

**Reply by rizkiandrianto** [reply]
Yes @gilbarbara , but as you can see, looks like it replacing the `<meta name="description"...` i gave it before.
Do you know why this can be happened?

**Reply by gilbarbara** [reply]
I have no idea.
